examples/printerdemo_mcu/esp-idf/main/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+
+idf_component_register(
+    SRCS "main.cpp"
+    INCLUDE_DIRS "."
+    REQUIRES slint esp-box
+    )
+
+
+slint_target_sources(${COMPONENT_LIB} ../../ui/printerdemo.slint)
+target_link_options(${COMPONENT_LIB} PUBLIC -Wl,--allow-multiple-definition)
+set_target_properties(${COMPONENT_LIB} PROPERTIES SLINT_EMBED_RESOURCES "embed-for-software-renderer")
+
+
+
```

Comment:
This file could use some reformatting (stray newlines, line 9 indentation seems off :)